File: JanperCab/3-Application/Dtos/DuraformDesign/DuraformDesignForOrderMenu.cs
using System.Collections.ObjectModel;

namespace _3_Application.Dtos.DuraformDesign
{
    public class DuraformDesignForOrderMenu : DuraformDesignDto
    {
        public string DefaultEdgeProfileName { get; set; }

        public Collection<DuraformDesignEdgeProfileDto> AllowedEdgeProfiles { set; get; }
    }
}

File: JanperCab/5-JanperCab/ClientApp/src/app/_models/duraform-design/DuraformDesignForOrderMenu.ts
import { DuraformDesignDto } from './DuraformDesignDto';
import { DuraformDesignEdgeProfileDto } from './DuraformDesignEdgeProfileDto';

export class DuraformDesignForOrderMenu extends DuraformDesignDto {
  defaultEdgeProfileName: string;
  allowedEdgeProfiles: DuraformDesignEdgeProfileDto[] = [];
}
